agents/check_mk_agent.linux

```diff
@@ -439,7 +439,7 @@ section_mem() {
 }
 
 section_cpu() {
-    if [ "$(uname -m)" = "armv7l" ] || [ "$(uname -m)" = "armv6l" ]; then
+    if [ "$(uname -m)" = "armv7l" ] || [ "$(uname -m)" = "armv6l" ] || [ "$(uname -m)" = "aarch64" ]; then
```

Comment:
Quick drive-by comment: When we have a 3rd condition, using a `case` might be more appropriate. It's more readable and more efficient (only a single uname call).